src/utility/include/mp-units/math.h

```diff
@@ -136,6 +136,41 @@ template<auto R, typename Rep>
   return {static_cast<Rep>(abs(q.numerical_value_ref_in(q.unit))), R};
 }
 
+/**
+ * @brief Computes the fma of 3 quantities
+ *
+ * @param a: Multiplicand
+ * @param x: Multiplicand
+ * @param b: Addend
+ * @return Quantity: The nearest floating point representable to ax+b
+ */
+template<auto R, auto S, auto T, typename Rep1, typename Rep2, typename Rep3>
+[[nodiscard]] constexpr QuantityOf<common_quantity_spec(get_quantity_spec(R) * get_quantity_spec(S),
+                                                        get_quantity_spec(T))> auto
+fma(  // QuantityOf takes a QuantitySpec as an argument
+  const quantity<R, Rep1>& a, const quantity<S, Rep2>& x, const quantity<T, Rep3>& b) noexcept
+  requires requires {
+    common_quantity_spec(get_quantity_spec(R) * get_quantity_spec(S), get_quantity_spec(T));
+  } &&  // or just common_reference but I wanted to express that the quantity types must match here and we don't care
```

Comment:
I would recommend either removing those comments or if you think such information should remain they should be refactored to sound more professional and possibly be put as a part of a function documentation above.